docs/css/mkdocstrings.css

```diff
@@ -40,43 +40,18 @@ h5.doc-heading {
   text-transform: none !important;
 }
 
-.md-typeset h4:target::before {
-  display: block;
-  margin-top: -4.1rem;
-  padding-top: 4.1rem;
-  content: "";
-}
-
-h1 {
-  margin: 0 !important;
-}
+/* Additional Customizations */
 
-h2.doc.doc-heading {
-  font-size: 1em !important;
-}
+/* Increase heading margins & weight. */
+.doc > h3 { margin: 3em 0 .75em 0; }
+.md-typeset h3 { font-weight: 700; }
+.md-typeset h5 { margin-top: 3em; }
 
-h2.doc.doc-heading > code {
-  font-size: 1.5625em !important;
-  vertical-align: baseline !important;
+/* Increased spacing between macro & instance methods. */
+.doc-macro + .doc-macro,
+.doc-instance_method + .doc-instance_method {
+  margin-top: 3em;
 }
 
-h2.doc.doc-heading > small {
-  font-size: 90% !important;
-}
-
-.doc.doc-object {
-  font-size: .85rem !important;
-}
-
-.doc.doc-method {
-  padding-bottom: 0.5em;
-}
-
-.doc-method h4 {
-  padding: 10px !important;
-  background-color: #F8F8F8 !important;
```

Comment:
Just for reference, I found this color to be an exact replacement for `#F8F8F8` that still works in dark mode:
`rgba(192, 192, 192, 0.10)`